### PR TITLE
Displayed url of image origin, in MapKnitter Lite interface

### PR DESCRIPTION
**Pull request by 7malikk**
### Screenshot
image-Url-UI

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `grunt test`
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [x] screenshots/GIFs are attached 📎 in case of UI updates
* [x] @mention the original creator of the issue in a comment below for help or for a review

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

**Comment by 7malikk** [comment]
@jywarren @TildaDares @cesswairimu Hello, kindly review my PR, it fixes [1153](https://github.com/publiclab/Leaflet.DistortableImage/issues/1153)
Thank you.

File: examples/archive.html [hunk 1]
```diff
@@ -2,61 +2,131 @@
...
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+      integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC"
+      crossorigin="anonymous"
+    />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+
+    <script
```
Comment (jywarren) [inline examples/archive.html:30]: Hmm it looks like your editor is adding formatting. Would you mind excluding these?

File: examples/archive.html [hunk 2]
```diff
@@ -91,39 +161,63 @@ <h3 id="offcanvasRightLabel">Images</h3>
...
+                );
                 imageRow.append(image, placeButton);
 
                 imageContainer.appendChild(imageRow);
                 imageCount++;
               }
             });
-            responseText.innerHTML = imageCount ? `${imageCount} image(s) fetched successfully.` : 'No images found in the link provided...'
+            responseText.innerHTML = imageCount
+              ? `${imageCount} image(s) fetched successfully ${fetchedFrom.innerHTML}.`
```
Comment (jywarren) [inline examples/archive.html]: I think we may need the word "from" here? And if we are adding it to the same line, perhaps we don't need to make a new <p> ? Can you share a screenshot of how it looks? Thank you!

**Review by jywarren — CHANGES_REQUESTED** [review]

**Comment by jywarren** [comment]
Great start here!

File: examples/archive.html (hunk 2 above)
Comment (7malikk) [reply examples/archive.html]: Sure, just not caught that.

Pardon me, I don't understand what you meant by "we don't need to make a new?" could you clarify

Sure, I'll attach a screenshot right away.

File: examples/archive.html (hunk 1 above)
Comment (7malikk) [reply examples/archive.html:30]: Will do

**Comment by 7malikk** [comment]
@jywarren Hello, i have added a screenshot and excluding the code editor fromatting

**Comment by 7malikk** [comment]
Hello, @TildaDares,could you review my PR

**Comment by 7malikk** [comment]
@TildaDares kindly review my PR

**Comment by 7malikk** [comment]
Hello @jywarren could you review my PR, thank you!

File: examples/archive.html [hunk 3]
```diff
@@ -1,65 +1,135 @@
...
+    <meta
+      name='viewport'
+      content='width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no'
+    />
+    <meta http-equiv='x-ua-compatible' content='ie=edge' />
+    <link rel='stylesheet' href='../node_modules/leaflet/dist/leaflet.css' />
+    <link rel='stylesheet' href='../dist/vendor.css' />
+    <link
+      rel='stylesheet'
+      href='../dist/leaflet.distortableimage.css'
```
Comment (jywarren) [inline examples/archive.html:15]: Hmm it still looks like there are new lines and spaces added here. Could you leave any unrelated lines as they were? Thank you so much!

Comment (7malikk) [reply examples/archive.html:15]: Hello @jywarren I'll get on it, in the mean time could you check [this out](https://github.com/publiclab/Leaflet.DistortableImage/issues/665), i and @THEDARKMAN007 need some information. 
Thank you!

**Comment by 7malikk** [comment]
@jywarren I've removed the spaces and new lines, kindly review.
Thanks a bunch

**Comment by jywarren** [comment]
Looking good - i still see changes from ' to " - can you revert those too? Basically in this view, it's very helpful to see ONLY the changes relevant to the issue: https://github.com/publiclab/Leaflet.DistortableImage/pull/1154/files

File: examples/archive.html [hunk 4]
```diff
@@ -91,39 +90,60 @@ <h3 id="offcanvasRightLabel">Images</h3>
...
                 let imageRow = document.createElement('div');
                 let image = new Image(150, 150);
                 let placeButton = document.createElement('a');
-
-                placeButton.classList.add('btn', 'btn-sm', 'btn-outline-secondary', 'place-button');
+                fetchedFrom = document.createElement('p');
+                fetchedFromUrl = document.createElement('a');
+                fetchedFromUrl.setAttribute('href', input.value);
+                fetchedFromUrl.setAttribute('target', '_blank');
+                fetchedFromUrl.innerHTML = 'here';
```
Comment (jywarren) [inline examples/archive.html:112]: Otherwise looking great! I'll make this small suggestion:

```suggestion
                fetchedFromUrl.innerHTML = 'this Internet Archive collection';
```

**Comment by 7malikk** [comment]
Alright, @jywarren inlcuded your impeccable suggestion. Ready for a review and merge

File: examples/archive.html [hunk 5]
```diff
@@ -1,65 +1,64 @@
 <!DOCTYPE html>
-<html lang="en">
```
Comment (jywarren) [inline examples/archive.html:2]: Ah, can you please take another look at https://github.com/publiclab/Leaflet.DistortableImage/pull/1154/files to see the double/single quotation mark changes that are still needed? Thank you!

Comment (7malikk) [reply examples/archive.html:2]: On it @jywarren

**Comment by 7malikk** [comment]
@jywarren, You must be tired of my pings but I promise this is the last one 😆

**Comment by jywarren** [comment]
Haha no worries! Actually there are still a few unrelated formatting differences in https://github.com/publiclab/Leaflet.DistortableImage/pull/1154/files - we really appreciate if your changes are ONLY those relevant to the feature in this PR. Sorry to keep asking for more changes!

**Comment by 7malikk** [comment]
Oh, okay, I'd get right on that @jywarren 😁

**Comment by 7malikk** [comment]
@jywarren final one i promise 🤣

**Comment by jywarren** [comment]
Hi, I just reverted all the changes in the dist folder, because you didn't change anything in /src/ so there shouldn't be anything compiled in /dist/ -- probably it had auto-built and got some slight version differences in your dependencies, but we should be fine now! Will merge as soon as the test passes, great work!

**Comment by jywarren** [comment]
🎉

**Comment by jywarren** [comment]
great job!!!

**Comment by 7malikk** [comment]
Thank you so much @jywarren  
It's been a long time coming😁
More to come by the way😌
